.gitlab/templates/detect-global-locks.yml

```diff
@@ -0,0 +1,23 @@
+detect-global-locks:
+  extends: .cached_testrunner
+  stage: setup
+  needs: []
+  parallel:
+    matrix:
+      - PYTHON_VERSION: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
+  variables:
+    DD_DYNAMIC_INSTRUMENTATION_ENABLED: '1'
+    DD_CODE_ORIGIN_FOR_SPANS_ENABLED: '1'
+    DD_EXCEPTION_REPLAY_ENABLED: '1'
+    DD_APPSEC_ENABLED: '1'
+    DD_APPSEC_SCA_ENABLED: '1'
+    DD_IAST_ENABLED: '1'
+    DD_RUNTIME_METRICS_ENABLED: '1'
+    DD_PROFILING_ENABLED: '1'
+    DD_PROFILING_LOCK_ENABLED: '0' # This patches the lock class
```

Comment:
```suggestion
    DD_PROFILING_LOCK_ENABLED: '0' # This patches the lock class
    DD_REMOTE_CONFIGURATION_ENABLED: '1'
```

?

Reply:
👍  DI force-enables RC, but it's an implementation detail so probably worth doing this explicitly.